### ISSUE TITLE
代码高亮好像问题

### ISSUE THREAD

**Issue by Windmzx** [yanm1ng/hexo-theme-vexo#10]
rt，似乎在代码中空行会在均匀的显示在代码前后，而不是空行原本应该在地方。

**Comment by yanm1ng** [comment]
你可以发一下截图吗？

**Comment by Windmzx** [comment]
![](http://blogstatic.mengzhexin.com/17-6-26/57851882.jpg)

**Comment by yanm1ng** [comment]
我这边并没有这个问题诶，这个比较难复现。。。你把 `config.yml` 里highlight的 `line_number` 设置为 `false` 试试

**Comment by Windmzx** [comment]
去掉行号，空号不会在两端显示。

我使用2个空格加一个回车作为空行，但是总是下面的的空号生效。（如果只添加第一个，则空行在第一处，添加第二处则只有第二处生效，添加第三次 只有第三处生效）
![](http://blogstatic.mengzhexin.com/17-6-26/42559704.jpg)

**Comment by Windmzx** [comment]
http://blogstatic.mengzhexin.com/17-6-26/42559704.jpg 图片似乎有问题

**Comment by yanm1ng** [comment]
这可能要看一下 highlight.js 的文档有没有说明了。

**Comment by ky0ncheng** [comment]
@Windmzx 
This page isn’t working

blogstatic.mengzhexin.com didn’t send any data.
ERR_EMPTY_RESPONSE

**Comment by sshuair** [comment]
代码高亮确实有问题，比如我这里的python代码，除了line number有问题外，代码颜色也是有问题

http://sshuair.com/2017/10/21/pytorch-loss/#CrossEntropyLoss

![2017-10-22 5 20 03](https://user-images.githubusercontent.com/5787131/31860494-5a717142-b6e0-11e7-9a1f-4bc157a44582.png)
